deploy/helm/values.yaml

```diff
@@ -629,3 +657,197 @@ ebpfNetworkMonitoring:
       repository: ""
       tag: ""
       pullPolicy: IfNotPresent
+
+prometheus:  
```

Comment:
I would explicitly comment here that this section is needed only when opencost is enabled otherwise is ignored.